ai-catalog/ui-component/src/main/java/com/android/ai/uicomponent/Buttons.kt

```diff
@@ -291,3 +293,36 @@ fun BackButtonPreview() {
         )
     }
 }
+
+@OptIn(ExperimentalMaterial3ExpressiveApi::class)
+@Composable
+fun UndoButton(
```

Comment:
@lethargicpanda I added the `Undo` button here to distinguish it from `Back` button to follow the design mocks